rskj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -155,6 +155,19 @@ private boolean init() {
             return true;
         }
 
+        if (isInitCodeSizeInvalidForTx(tx)) {
+
+            String errorMessage = String.format("Initcode size for contract is invalid, it exceed the max limit size: initcode size = %d | maxAllowed = %d |  tx = %s", getLength(tx.getData()), Constants.getMaxInitCodeSize(), tx.getHash());
+
+            logger.warn(errorMessage);
+            logger.warn("Transaction Data: {}", tx);
+            logger.warn("Tx Included in the following block: {}", this.executionBlock);
```

Comment:
curious if this can be done in one logger.warn() call? Also I wouldn't log tx / executionBlock objects, as they are user generated data and can be quite big. You already log tx hash in errorMessage.